packages/labs/react/package.json

```diff
@@ -74,9 +55,7 @@
     "@types/react-dom": "^16.9.8"
   },
   "dependencies": {
-    "@lit/reactive-element": "^1.0.0-pre.1",
-    "lit-element": "^3.0.0-pre.2",
-    "lit-html": "^2.0.0-pre.4"
+    "@lit/reactive-element": "^1.0.0-pre.1"
```

Comment:
Should be a `devDependency` I think? Not imported in `create-component.ts`

Reply:
Done.